packet_transport.go

```diff
@@ -0,0 +1,15 @@
+package swim
+
+// PacketTransportConfig is used to configure a udp transport
+type PacketTransportConfig struct {
+	// BindAddrs is representing list of address to use for  UDP communication
+	BindAddress string
+
+	// BindPort is the port to listen on. for each address specified above
+	BindPort int
```

Comment:
// BindPort is the port to listen on. for each address specified above
-> // BindPort is the port to listen on. for the address specified above